### PR TITLE
[Enhancement] Add host start time info on BE start

### PR DESCRIPTION
**Pull request by decster**
## Why I'm doing:

When BE restarts, it's hard to know whether process restarts or  host restarts

## What I'm doing:

Adding server start time in be.out so it's easier to infer this information

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.3
  - [x] 3.2
  - [x] 3.1
  - [x] 3.0
  - [x] 2.5

### CODE REVIEW

File: bin/start_backend.sh [hunk 1]
```diff
@@ -206,7 +206,7 @@ else
     exec &>> ${LOG_FILE}
 fi
 
-echo "start time: "$(date)
+echo "start time: "$(date)" server:"$(uptime)
```
Comment (imay) [inline bin/start_backend.sh:209]:
```suggestion
echo "start time: "$(date)", server uptime: "$(uptime)
```

**Review by imay — COMMENTED** [review]
Do you think it is necessary to add uptime when FE starts?

File: bin/start_backend.sh (hunk 1 above)
Comment (kevincai) [inline bin/start_backend.sh:209]:
```suggestion
echo "start time: $(date), server:$(uptime)"
```

Comment (kevincai) [reply bin/start_backend.sh:209]: it is ok to include $() inside "", shell will interpret it correctly.

Comment (decster) [reply bin/start_backend.sh]: done

**Review by kevincai — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Pull request merged**